pkgs/development/python-modules/thumbor/default.nix

```diff
@@ -46,6 +46,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   meta = with stdenv.lib; {
+    broken = isPy3k; # see https://github.com/thumbor/thumbor/issues/1004
```

Comment:
shouldn’t we use the disabled property? This gives an error message which is way more helpful

Reply:
Yes, we use `disabled`.

Reply:
Sure, updated.